app/views/organizations/adoptable_pets/show.html.erb

```diff
@@ -102,8 +102,8 @@
                 <% else %>
                   <% if AdopterApplication.adoption_exists?(current_user.adopter_foster_account&.id, @pet.id) %>
                     <div class='d-flex align-items-center mt-3'>
-                      <h4 class="me-2">
-                        <%= "#{@adoption_application.human_enum_name(@adoption_application.status) || t('.status.default')}" %>
+                      <h4 class="me-2 mb-0">
+                        <%= "Application Status: #{@adoption_application.human_enum_name(:status) || t('.status.default')}" %>
```

Comment:
Looks like you prepended "Application Status:" for clarity. Do we have a translation you could use there as well?

Reply:
Haha classic I use a string in a view while working on translation fix. I will fix that.